src/pages/blog/leaf-is-five-years-old.pug

```diff
@@ -0,0 +1,112 @@
+
+            p.blog-post__text Dedicating your time to intimately understanding a single area of interest creates an intense relevance with a much smaller subset of potential clients, who themselves are in need of expert advice within that one specific subject matter.
+
+            p.blog-post__text What's more, Leaf will always be a small studio. We may get a little bigger or a little smaller over time, but we're not shooting for sharp or extreme growth. As such, we're only ever looking for a handful of perfectly suited clients; it's an anti-conveyor-belt, atypical agency model, and it works great for us.
+
+            h3.blog-post__h3 Make decisions, even if they're imperfect ones
+
+            p.blog-post__text It's only natural that you'd want all the facts before making any kind of significant commitment, but it's rarely an efficient way to run a business. To avoid bottlenecks, key decision makers have to be comfortable with committing to judgement calls based on what they <em>do</em> know.
+
+            p.blog-post__text It's unlikely that a decision made from your experience, your gut, and _some_ evidence will be woefully wrong. It may be less-than-perfect, but achieving 80% of the value without weeks, months, or even years of exploration, research, tests, and discussions is infinitely more desirable than being paralysed by planning.
+
+            h3.blog-post__h3 Work on the business, not in the business
+
+            p.blog-post__text Three years back, Mike and I spent pretty much every working minute solving client problems, rather than our own. This approach is exciting, and the coffers quickly swell, but your growth is capped, and the risk to your business should you lose a few clients is considerable.
+
+            p.blog-post__text We've still not won this battle, but don't need to be convinced of the benefit. More than half of our time is now spent working <em>on</em> Leaf, rather than <em>in</em> Leaf. Managing people, clients, sales funnels, and marketing strategies takes time, and isn't optional.
+
+            p.blog-post__text Build yourself a team capable of doing great work, and support them with the structure and an environment that lets them do just that.
+
+            h3.blog-post__h3 Delegate where possible
+
+            p.blog-post__text We're surrounded by talented, highly skilled individuals, who are able to take a portion of our workloads off our plate, but so often we don't feel comfortable delegating.
+
+            p.blog-post__text By taking ownership of every detail, you not only spread yourself too thinly, but you create unnecessary bottlenecks. Your skills may seen considerable, but like your time, they only go so far.
+
+            p.blog-post__text Learn to let go, and trust in others to not completely screw it up. They might even surprise you.
```

Comment:
I think the last sentence here could be misinterpreted as not being tongue and cheek.